code/game/objects/structures/manned_turret.dm

```diff
@@ -38,33 +38,34 @@
 		buckled_mob.pixel_x = 0
 		buckled_mob.pixel_y = 0
 		if(buckled_mob.client)
-			buckled_mob.reset_perspective()
+			buckled_mob.client.change_view(world.view)

```

Comment:
@Cyberboss perspective refers to the ```perspective``` variable of ```/client```, and of its eye object.
It has no relation to view size.